packages/expo-dev-menu/ios/Modules/DevMenuPreferences.swift

```diff
@@ -84,8 +84,10 @@ public class DevMenuPreferences: NSObject, RCTBridgeModule {
 
   /**
    Returns `true` only if the user finished onboarding, `false` otherwise.
+   This is now public because expo-dev-launcher needs access
```

Comment:
```suggestion
   This is public because expo-dev-launcher needs access
```

Reply:
@ajsmth I am actively working on the docs for this -- had a discussion with @brentvatne yesterday and we are attempting to figure out exactly where this should be in the docs ("Development Builds" and/or "EAS E2E tests").

I'll go ahead and land this and create a separate PR for the doc changes.